resource/ebpf_version.h

```diff
@@ -5,5 +5,9 @@
 #define EBPF_VERSION_MINOR 7
```

Comment:
Shouldn't this be 8? Also, going forward, I'm assuming this would be manually updated for each new minor version release?

Reply:
Agree. I think whenever we update version in a release branch, we should also update the version in the main branch.

Reply:
Agreed.